index.html

```diff
@@ -76,6 +76,6 @@
 
 <h2>Workers</h2>
 <ul>
-  <li><a href="worker/webworker.html">Web Workers</a></li>
-  <li><a href="worker/push.html">Push API with Service Worker</a></li>
-</ul>
\ No newline at end of file
+  <li><a href="workers/hello-workers/webworker.html">Web Workers</a></li>
```

Comment:
My 2 cents: I'd rather keep the URLs for the HTML pages stable and then have them reference scripts / workers from the subdirectories.  Mainly just to keep the old URLs working since they are referenced across a number of bugs.  Totally on board with moving sub resources into subfolders for better organization though